librz/include/rz_types.h

```diff
@@ -676,8 +676,8 @@ static inline void rz_run_call10(void *fcn, void *arg1, void *arg2, void *arg3,
 }
 
 #define RZ_V_NOT(op, fail_ret) \
-	if ((op) == (fail_ret)) \
-		eprintf (#op" at %s:%d failed: %s\n", __FILE__, __LINE__, strerror (errno))
+	if ((op) != (fail_ret)) \
```

Comment:
??

Reply:
It's just to see whether the CI captures RZ_LOG_WARN output ... it has been reverted don't worry